db/schema.js

```diff
@@ -1,4 +1,40 @@
 var mongoose = require('mongoose');
 mongoose.connect('mongodb://localhost/yum');
```

Comment:
I would move this line right before you export `mongoose` and after you define your models